src/controllers/processors/EvidenceUploadFormSubmissionProcessor.ts

```diff
@@ -0,0 +1,77 @@
+import * as Busboy from 'busboy';
+import { Request } from 'express';
+import { inject } from 'inversify';
+import { provide } from 'inversify-binding-decorators';
+import { Socket } from 'net';
+
+import { FormSubmissionProcessor } from 'app/controllers/processors/FormSubmissionProcessor';
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData, APPLICATION_DATA_KEY } from 'app/models/ApplicationData';
+import { EvidenceUploadService } from 'app/service/EvidenceUploadService';
+
+@provide(EvidenceUploadFormSubmissionProcessor)
+export class EvidenceUploadFormSubmissionProcessor implements FormSubmissionProcessor {
+
+    constructor(@inject(EvidenceUploadService) private readonly evidenceUploadService: EvidenceUploadService) {
+    }
+
+    async process(req: Request): Promise<void> {
+
+        const id = '';
+
+        console.log(req.headers['content-type']);
+
+        const appeal: Appeal = req.session
+            .chain(_ => _.getExtraData())
+            .map<ApplicationData>(data => data[APPLICATION_DATA_KEY])
+            .map(data => data.appeal)
+            .unsafeCoerce();
+
+        console.log(`Uploading file for company number: ${appeal.penaltyIdentifier.companyNumber}` +
```

Comment:
Let's use CH logger for any logs that will remain in code base for troubleshooting. I am OK for us to use `console.log` temporarily but PR won't be approved until all `console.log` call are eradicated.